sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/implementation/util/Transforms.java

```diff
@@ -270,6 +267,14 @@ public static Throwable mapToHttpResponseExceptionIfExists(Throwable throwable)
         return throwable;
     }
 
+    public static HttpResponseException mapResponseErrorToHttpResponseException(Error error) {
+        return new HttpResponseException(
+            error.getMessage(),
+            null,
```

Comment:
This should not be `null`. We should include the HttpResponse here as that contains useful details like HttpRequest, the response headers and the HTTP status code.

Reply:
This is a client-side error being thrown as an exception and we do not have a way to populate HttpResponse here.